src/learning-header/AuthenticatedUserDropdown.jsx

```diff
@@ -16,10 +16,12 @@ const AuthenticatedUserDropdown = ({ intl, username }) => {
       {intl.formatMessage(messages.dashboard)}
     </Dropdown.Item>
   );
-
+  const SupportUrl = (
+    <a className="text-gray-700 mr-3" href={`${getConfig().SUPPORT_URL}`}>{intl.formatMessage(messages.help)}</a>
+  );
```

Comment:
I don't think we need to specify a separate variable for this.  It's fine if we just do it  below, via:

```
{getConfig().SUPPORT_URL && (
  <a>...</a>
)}